lib/google_crawler/accounts/user.ex

```diff
@@ -0,0 +1,35 @@
+defmodule GoogleCrawler.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "users" do
+    field :email, :string
+    field :username, :string
+    field :encrypted_password, :string
+
+    field :password, :string, virtual: true
+    field :password_confirmation, :string, virtual: true
+
+    timestamps()
+  end
+
+  @fields ~w(email username encrypted_password password password_confirmation)a
```

Comment:
Good idea to move to a constant 👍

Reply:
I saw some people doing this 😂 

Actually I saw they do

```
@required_fields
@optional_fields
```
But I am not sure, so I just put only one for now to reduce the long arguments in the cast field 💭